tcti/platformcommand.c

```diff
@@ -62,7 +62,7 @@ TSS2_RC PlatformCommand(
     // Send the command
     iResult = send( TCTI_CONTEXT_INTEL->otherSock, sendbuf, 4, 0 );
     if (iResult == SOCKET_ERROR) {
-        (*printfFunction)(NO_PREFIX, "send failed with error: %d\n", WSAGetLastError() );
+        DebugPrintf(NO_PREFIX, "send failed with error: %d\n", WSAGetLastError() );
```

Comment:
I think this change is an artifact from a rebase? I've got an outstanding PR that gets ride of this function pointer so maybe this snuck in when you were reviewing / testing that work?